server/routes.go

```diff
@@ -70,6 +70,12 @@ func routes(h Handler, conf *config.Config) Routes {
 			Handler: h.GetFromSandbox("stderr"),
 		},
 		Route{
+			Name:    "Kill",
+			Method:  "POST",
```

Comment:
I believe `DELETE` can be used even if you're doing a soft delete on the resource, still being available for inspection by an admin (i.e. I could still be in the database). Although I think you're right about that in that case it shouldn't be available on `GET /task/{taskId}`. I'm fine with this.